docs/intro/reusable-apps.txt

```diff
@@ -242,8 +242,8 @@ this. For a small app like polls, this process isn't too difficult.
 #. Only Python modules and packages are included in the package by default. To
    include additional files, we'll need to create a ``MANIFEST.in`` file. The
    setuptools docs referred to in the previous step discuss this file in more
-   detail. To include the templates, the ``README.rst`` and our ``LICENSE``
-   file, create a file ``django-polls/MANIFEST.in`` with the following
+   detail. To include the static files, the template files, the ``README.rst`` and ``LICENSE``
```

Comment:
please wrap line length
Also I think it could be a little more concise:
```
... the templates, static files, ``README.rst``, and ``LICENSE``, 
```
(It's a little confusing that this change isn't related, so probably it should be a separate commit.)